src/main/java/lt/vu/persistence/orm/entities/Attribute.java

```diff
@@ -0,0 +1,43 @@
+package lt.vu.persistence.orm.entities;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Entity
+@Table(name = "ATTRIBUTE")
+@NamedQueries({
+    @NamedQuery(name = "Attribute.findAll", query = "select a from Attribute a"),
+    @NamedQuery(name = "Attribute.findAllByIds", query = "select a from Attribute a where a.id in (:ids)"),
+})
+@Getter @Setter
+public class Attribute implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name = "LABEL", nullable = false)
+    private String label;
```

Comment:
Why do we need this `label` field? What's the difference between label and `type`?

Reply:
Label is meant for frontend. Type is for internal use in backend to identify these attributes by purpose (for example calculating prices etc.)